taxonomies/categories.txt

```diff
@@ -220,6 +220,7 @@ ciqual_food_name:fr:Boisson à base de plantes pour bébé
 <en:Baby drinks
 en:Milky cereal-based beverages, Milky cereal-based beverage
 fr:Boisson infantile lactée aux céréales, Boisson infantile lactée aux céréales
+agribalyse_proxy_food_code:en:13173
```

Comment:
please also add agribalyse_proxy_food_name so that it's easier to see what proxy has been used